resources/dns/records.tf

```diff
@@ -75,7 +71,40 @@ resource "cloudflare_record" "uptime" {
   ttl     = 1
   type    = "A"
   value   = "35.89.37.220"
-  zone_id = data.cloudflare_zone.cloudflare_zone.id
+  zone_id = data.cloudflare_zone.subspace_network.id
+}
+
+// subspace Grafana monitor
+resource "cloudflare_record" "grafana" {
```

Comment:
If we have grafana exposed, do we have the auth setup on grafana ?

Reply:
https://grafana.subspace.network/